Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -6736,4 +6736,68 @@ final class BuildPlanTests: XCTestCase {
             )
         }
     }
+
+    func testHostTargetDontCross() async throws {
```

Comment:
naming nit: `Cross` could refer to cross compilation here, but I thought we're actually referring to overlapping builds?

```suggestion
    func testHostTargetDontOverlap() async throws {
```

Reply:
It's referring to crossing the streams. Been watching too much Ghostbusters lately :p.

Reply:
Renamed to better reflect.